src/frames.c

```diff
@@ -307,8 +307,9 @@ static int is_frame_initialized(const novas_frame *frame) {
 int novas_make_frame(enum novas_accuracy accuracy, const observer *obs, const novas_timespec *time, double dx, double dy,
         novas_frame *frame) {
   static const char *fn = "novas_create_frame";
-  static const object earth = { NOVAS_PLANET, NOVAS_EARTH, "Earth" };
-  static const object sun = { NOVAS_PLANET, NOVAS_SUN, "Sun" };
+  static const cat_entry zero_star = {0};
```

Comment:
Is it in response to a compiler warning? Otherwise, I'm not sure this is necessary. You are correct that the initializer needs to cover at least one element (although gcc seems to not require even that), but after the defined elements, all the remaining footprint of the structure is initialized to 0, as if by `memset()`. So, there should be no need to explicitly define null substructures for the initializer. I think it's just unnecessarily verbose, but I might be wrong...